Userland/Utilities/groups.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <grp.h>
+#include <pwd.h>
+#include <unistd.h>
+
+#include <AK/Vector.h>
+#include <LibCore/Account.h>
+#include <LibCore/ArgsParser.h>
```

Comment:
no need to separate these.

```suggestion
#include <AK/Vector.h>
#include <LibCore/Account.h>
#include <LibCore/ArgsParser.h>
#include <grp.h>
#include <pwd.h>
#include <unistd.h>
```